components/support/base/src/main/java/mozilla/components/support/base/observer/Consumable.kt

```diff
@@ -4,16 +4,13 @@
 
 package mozilla.components.support.base.observer
 
-import android.support.annotation.VisibleForTesting
-
 /**
  * A generic wrapper for values that can get consumed.
  */
 class Consumable<T> private constructor(
     private var value: T?
 ) {
-    // Internal getter for testing only.
-    @VisibleForTesting internal val internalValue: T?
+    internal val internalValue: T?
         get() = value
```

Comment:
Now that we use this internally we could just make the `value` property itself internal.

Reply:
True, I will update this.